src/contour/opengl/OpenGLRenderer.h

```diff
@@ -18,9 +18,9 @@
 
 #include <crispy/debuglog.h>
 
-#include <QtGui/QMatrix4x4>
-#include <QtGui/QOpenGLExtraFunctions>
-#include <QtGui/QOpenGLShaderProgram>
+#include <QMatrix4x4>
```

Comment:
Why doesn't this work anymore? (basically the same question as above :) )

Reply:
Oh!
Don't now why I changed it. :(